instructors.md

```diff
@@ -7,196 +7,245 @@ subtitle: Instructor's Guide
+    the end using `git config --list`.
 
 ## [Creating a Repository](03-create.html)
 
-The challenge "Places to create repositories" tries to reinforce the idea that
-the `.git` folder contains the whole Git repo and deleting this folder *undoes*
-a `git init`. It also gives the learner the way to fix the common mistake of
-putting unwanted folders (like `Desktop`) under version control.
+*   When you do `git status`, Mac users may see a `.DS_Store` file showing as 
+    untracked. This a file that Mac OS creates in each directory.
+
+*   The challenge "Places to create repositories" tries to reinforce the idea
+    that the `.git` folder contains the whole Git repo and deleting this folder
+    undoes a `git init`. It also gives the learner the way to fix the common
+    mistake of putting unwanted folders (like `Desktop`) under version control.
+    
+    Instead of removing the `.git` folder directly, you can choose to move it
+    first to a safer directory and remove it from there:
+    
+    ~~~{.input} 
+    $ mv .git temp_git 
+    $ rm -rf  temp_git 
+    ~~~
 
-Instead of removing the `.git` folder directly, you can choose to move
-it first to a safer directory and remove it from there:
+## [Tracking Changes](04-changes.html)
 
-~~~ {.bash}
-mv .git temp_git
-rm -rf  temp_git
-~~~
+*   It's important that learners do a full commit cycle by themselves (make
+    changes, `git add`, `git diff`, and `git commit`). The "`bio` repository"
+    challenge does that.
 
-## [Tracking Changes](04-changes.html)
+*   This is a good moment to show a diff with a graphical diff tool. If you 
+    skip it because you're short on time, show it once in Github.
 
 ## [Exploring History](05-history.html)
 
-*   One thing that has caused confusion comes up when recovering old versions
-    towards the end of ["Exploring history"](04-history.html).
-    If, instead of doing `$ git checkout f22b25e mars.txt`,
-    someone does `$ git checkout f22b25e`,
-    they wind up in the "detached HEAD" state and confusion abounds.
-    It's then possible to keep on committing,
-    but things like `git push origin master` a bit later will not give easily comprehensible results.
-    It also makes it look like commits can be lost.
+*   One thing may cause confusion is recovering old versions.  If, instead of
+    doing `$ git checkout f22b25e mars.txt`, someone does `$ git checkout
+    f22b25e`, they wind up in the "detached HEAD" state and confusion abounds.
+    It's then possible to keep on committing, but things like `git push origin
+    master` a bit later will not give easily comprehensible results.  It also
+    makes it look like commits can be lost.  To "fix" a "detached HEAD", simply
+    `git checkout master`.
+
+*   This is a good moment to show a log within a Git GUI. If you skip it
+    because you're short on time, show it once in Github.
 
 ## [Ignoring Things](06-ignore.html)
+
+Just remember that you can use regular expressions in `.gitignore`.
```

Comment:
None, probably, but this is for the instructors. I'll change it to "wildcards and regular expressions to ignore a particular set of files". It's a common question whether or how you can ignore a particular set of files, say all executables in a `lib/` folder.